### PR TITLE
updating README + show_supported_models fix

### PR DESCRIPTION
**Pull request by thomas-davis**
Fix for https://github.com/nf-core/epitopeprediction/issues/224

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by thomas-davis** [comment]
Minor pt. Looks like https://github.com/nf-core/epitopeprediction/blob/master/bin/check_requested_models.py still refers to FRED2 (old name of Epytope?) in a couple of places, I can roll that fix into this PR as well

File: README.md [hunk 1]
```diff
@@ -16,6 +16,17 @@
 **nf-core/epitopeprediction** is a bioinformatics best-practice analysis pipeline for epitope prediction and annotation.
 The pipeline performs epitope predictions for a given set of variants or peptides directly using state of the art prediction tools. Additionally, resulting prediction results can be annotated with metadata.
 
+Supported prediction tools:
+
+- `syfpeithi`
```
Comment (christopher-mohr) [inline README.md:21]: we could also add references here (publication, website,...)

**Comment by christopher-mohr** [comment]
> Minor pt. Looks like https://github.com/nf-core/epitopeprediction/blob/master/bin/check_requested_models.py still refers to FRED2 (old name of Epytope?) in a couple of places, I can roll that fix into this PR as well

In which lines exactly? But in general, sure go ahead. :)

**Review by christopher-mohr — APPROVED** [review]
Thank you! LGTM, just one comment

**Comment by christopher-mohr** [comment]
@nf-core-bot fix linting

**Comment by christopher-mohr** [comment]
@nf-core-bot fix linting

**Pull request merged**